firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/Firestore.kt

```diff
@@ -18,15 +18,13 @@ import androidx.annotation.Keep
 import com.google.firebase.FirebaseApp
 import com.google.firebase.components.Component
 import com.google.firebase.components.ComponentRegistrar
-import com.google.firebase.firestore.DocumentSnapshot
-import com.google.firebase.firestore.FieldPath
-import com.google.firebase.firestore.FirebaseFirestore
-import com.google.firebase.firestore.QueryDocumentSnapshot
-import com.google.firebase.firestore.QuerySnapshot
-import com.google.firebase.firestore.FirebaseFirestoreSettings
+import com.google.firebase.firestore.*
```

Comment:
We generally avoid collapsing imports.

Reply:
I have reverted the collapsing imports

Reply:
@schmidt-sebastian could that configuration be commited in the repo (.idea folder or something else?)

Reply:
nit: this is not fixed apparently

Reply:
I think it got fixed initially, but reverted in a follow-up commit.

Reply:
Oh gosh, it's back... I just removed it again

Reply:
Yea, I'll do a follow up PR to commit the codeStyle so that AS stops doing this automagically. I "just" need to figure out how the .gitignore syntax works...

Reply:
Here it is: https://github.com/firebase/firebase-android-sdk/pull/3984